src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/UserRepository.kt

```diff
@@ -1,8 +1,7 @@
 package org.vaccineimpact.api.app.repositories
 
-import org.vaccineimpact.api.models.permissions.User
-
 interface UserRepository : Repository
 {
-    fun getUserByEmail(email: String): User?
+    fun getUserByEmail(email: String): org.vaccineimpact.api.security.MontaguUser?
+    fun getUserByUsername(username: String): org.vaccineimpact.api.models.User?
 }
```

Comment:
Now you've resolved the naming issue, you could go back to non-fully qualified class names here.